index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <h1 id="title">Tribute Page</h1>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://pbs.twimg.com/media/DGVBXfVV0AEyqXH.jpg"
+          alt="tribute-image"
+        />
+        <p id="img-caption">This is just an image</p>
+      </div>
+      <div id="tribute-info">
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Est,
+          perspiciatis.
+        </p>
+        <a
+          id="tribute-link"
+          href="https://en.wikipedia.org/wiki/Tribute_(website)"
+          target="_blank"
+          >View more</a
+        >
+      </div>
+    </main>
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Please don't include comments inside your document as this is considered a bad practice, kindly delete the
comments from your code as this is the best practice, if you have further comments you can add them in your
readme file. 😉✌🏻